help/dns_verification/dns_verification.rst

```diff
@@ -0,0 +1,87 @@
+.. _doc_dnsverification:
+.. _Dashboard: https://mailsac.com/dashboard
+.. _Domains: https://mailsac.com/domains
+
+DNS Verification
+================
+
+DNS (Domain Name System) is at the core of the internet and email delivery.
+When configuring a :ref:`BYO Domain <section_byod_configuration>` the following
+records need to be configured:
+
+- 2 MX Records
+- 1 DKIM Record
+- 1 SPF Record
+
+MX Record Verification
+----------------------
+
+The values of the MX Records are the :ref:`same <section_email_hosting>` for all
+Private and Public Domains hosted by Mailsac. The Host will be unique per
+domain.
+
+======== ================= ==================
+**Type** **Host**           **Value**
```

Comment:
If we're walking people all the way through these records, explaining what the `1` and `5` mean would be a good idea. I have seen registrars where they have you fill out a little form indicating the priority number, with the MX hostname separate.